app/components/atoms/socialMediaButtons/socialMediaButton.component.tsx

```diff
@@ -0,0 +1,22 @@
+import {FC} from 'react';
+import {ButtonProps, StyledButton} from './socialMediaButton.styles';
+import InstagramIconSVG from '../../../img/instagram-line.svg';
+import FacebookIconSVG from '../../../img/facebook-fill.svg';
+import TwitterIconSVG from '../../../img/twitter-fill.svg';
+
+
+export const Button: FC<ButtonProps> = ({ children, ...props}) => {
+  let icon;
```

Comment:
Q: `children` would be a string? 🤔 

why not using something like a function to return the button?

```typescript
const iconFromName = (name: String) => {
    switch(name) {
        case 'Instagram':
            return <img src={InstagramIconSVG} alt= "Instagram"/>
        case 'Facebook':
            return <img src={FacebookIconSVG} alt= "Facebook"/> 
        case 'Twitter':
            return <img src={TwitterIconSVG} alt= "Twitter"/> 
    }
}
```

then you can just use:

`return <StyledButton {...props}>{iconFromName(children)}</StyledButton>;`

feels weird to have `children` as the property name tho.